src/components/Articles.jsx

```diff
@@ -8,28 +8,58 @@ import { Link } from "react-router-dom"
 function Articles() {
```

Comment:
I see you've done ticket 9, but your topics page seems blank. This way I can't test your sorting while articles are selected by a category.